features/channel/managing_channels/excluding_chosen_taxons_from_showing_lowest_price_of_discounted_products.feature

```diff
@@ -9,19 +9,35 @@ Feature: Excluding chosen taxons from displaying the lowest price of discounted
         And the store classifies its products as "T-Shirts", "Caps" and "Sweaters"
         And I am logged in as an administrator
 
-    @api
+    @api @ui @javascript
     Scenario: Excluding a singular taxon from displaying the lowest price of discounted products
         When I want to modify a channel "Poland"
         And I exclude the "T-Shirts" taxon from showing the lowest price of discounted products
         And I save my changes
         Then I should be notified that it has been successfully edited
         And this channel should have "T-Shirts" taxon excluded from displaying the lowest price of discounted products
 
-    @api
+    @api @ui @javascript
     Scenario: Excluding multiple taxons from displaying the lowest price of discounted products
         When I want to modify a channel "Poland"
         And I exclude the "T-Shirts" and "Caps" taxons from showing the lowest price of discounted products
         And I save my changes
         Then I should be notified that it has been successfully edited
         And this channel should have "T-Shirts" taxon excluded from displaying the lowest price of discounted products
         And this channel should have "Caps" taxon excluded from displaying the lowest price of discounted products
+
+    @no-api @ui @javascript
+    Scenario: Remove excluded taxon from displaying the lowest price of discounted products
+        When I want to modify a channel "Poland"
+        And I exclude the "T-Shirts" taxon from showing the lowest price of discounted products
+        And I exclude the "Caps" taxon from showing the lowest price of discounted products
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this channel should have "T-Shirts" taxon excluded from displaying the lowest price of discounted products
+        And this channel should have "Caps" taxon excluded from displaying the lowest price of discounted products
+        When I want to modify a channel "Poland"
+        And I remove the "T-Shirts" taxon from excluded taxons from showing the lowest price of discounted products
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this channel should have "Caps" taxon excluded from displaying the lowest price of discounted products
+        And this channel should not have "T-Shirts" taxon excluded from displaying the lowest price of discounted products
```

Comment:
We shouldn't create a scenario that has `When -> Then -> When -> Then` steps. The initially excluded taxons should be added in the `Given` steps and only the second part of this scenario should be content.